src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  --main-color: #00acdc;
+  --background-color: #fff;
+  --text-secondary-color: #000;
+
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+  box-sizing: border-box;
+}
+  
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: var(--background-color);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header-logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  color: var(--text-secondary-color);
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__link:hover,
+.is-active {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  background-color: var(--main-color);
+  text-decoration: underline;
+  width: 37px;
```

```suggestion
  width: 100%;
```
This class should not depend on the predefined width, it should expand if needed